_layouts/quality_estimation.html

```diff
@@ -0,0 +1,124 @@
+  {% endif %}
+</p>
+
+
+<blockquote>
+
+  <dl>
+
+    <dt>Language support</dt>
+  {% if page.supported_languages.size == 0 %}
+    <dd><a href="#language-support">Unknown number of languages</a></dd>
+  {% else %}
+    <dd><a href="#language-support">{{ page.supported_languages | size }} languages</a></dd>
+  {% endif %}
+
+  
+  <dt>Integrations</dt>
+  {% if page.only_compatible_tms %}
+  <dd><a href="/{{ page.only_compatible_tms }}">{{ page.only_compatible_tms|capitalize }}</a></dd>
+  {% else %}
+  <dd>Any translation management system (API)</dd>
+  {% endif %}
+  
+  <dt>APIs</dt>
+  {% if page.only_compatible_mt_api %}
+  <dd><a href="/{{ page.only_compatible_mt_api }}">{{ page.only_compatible_mt_api|capitalize }}</a></dd>
+  {% else %}
+  <dd>Any machine translation API</dd>
+  {% endif %}
+</dl>
+
+  {% if page.urls.size > 0 %}
+    <hr/>
+    <ul>
+      {% for url in page.urls %}
+      <li>
+        <a href="{{ url }}" target="_blank">{{ url | remove_first: 'https://' | remove_first: 'http://' | remove_first: 'www.' | truncate: 50 }}</a>
+      </li>
+      {% endfor %}
+    </ul>
+  {% endif %}
+
+</blockquote>
+
+
+<section id="language-support">
+  <h2>Language support</h2>
+  {% if page.supported_languages.size > 0 %}
+
+    <details>
+      <summary>
+        {% if page.supported_languages_comment %}
+          {{ page.supported_languages_comment }}
+        {% endif %}
+        {% if page.supported_languages.size == 1 %}
+          <strong>1</strong> language is supported by the {{ page.title }} Company.
+        {% else %}
+          <strong>{{ page.supported_languages | size }}</strong> languages are supported by the {{ page.title }} Company.
+        {% endif %}
+        <p class="preview hint">
+          {{ page.supported_languages | slice: 0, 5 | map: 'name' | join: ', ' }}
+          {% if page.supported_languages.size > 5 %}, …{% endif %}
+        </p>
+      </summary>
+      <ul>
+        {% for language in page.supported_languages %}
+          <li>
+            {% if language.slug %}
+            <a href="/{{ language.slug }}">
+              <code>{{ language.base_code }}</code>
+              {{ language.name }}
+              {% if language.variant_name %}
+                | {{ language.variant_name }}
+              {% endif %}
+            </a>
+              {% if language.base_code != language.code %}
+                ⚠️ {{ page.title }} uses <code>{{ language.code }}</code>.
+              {% endif %}
+            {% else %}
+            <a href="https://en.wikipedia.org/wiki/ISO_639:{{ language.base_code }}" target="_blank">
+              <code>{{ language.code }}</code>
+              {% if language.variant_name %}
+                | {{ language.variant_name }}
+              {% endif %}
+            </a>
+            {% endif %}
+          </li>
+        {% endfor %}
+      </ul>
+    </details>
+
+{% else %}
+
+  <p>The {{ page.title }} company supports an unknown number of languages.</p>
```

Comment:
I'd change the sentence structure to:

´{{ page.title }} supports an unknown number of languages.´